ontology/occo.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: ontology_detail
+id: occo
+title: Occupation Ontology
+browsers:
+- title: BioPortal Ontology Browser
+  label: BioPortal
+  url: https://bioportal.bioontology.org/ontologies/OCCO?p=classes
+contact:
+  email: zhengj2007@gmail.com
+  github: zhengj2007
+  label: Jie Zheng
+  orcid: 0000-0002-2999-0103
+description: An ontology representing occupations. It is designed to facilitate harmonization of existing occupation standards, such as the US Bureau of Labor Statistics Standard Occupational Classification (US SOC), the International Standard Classification of Occupations (ISCO), the UK National Statistics Standard Occupational Classification (UK SOC), and the European Skills, Competences, Qualifications and Occupations (ESCO) of the European Union 2010.
```

Comment:
@zhengj2007 The description ends with "..and the European Skills, Competences, Qualifications and Occupations (ESCO) of the European Union 2010".

Not sure what the **2010** is indicating here.

Reply:
```suggestion
description: An ontology representing occupations. It is designed to facilitate harmonization of existing occupation standards, such as the US Bureau of Labor Statistics Standard Occupational Classification (US SOC), the International Standard Classification of Occupations (ISCO), the UK National Statistics Standard Occupational Classification (UK SOC), and the European Skills, Competences, Qualifications and Occupations (ESCO) of the European Union.
```

Reply:
The 2010 is the date since when ESCO has been developed which seems a copy-paste error here; Lets push this through so we can list it in the newsletter

Reply:
Thank you for the edit suggestion 👍